system/locales/ko.toml

```diff
@@ -0,0 +1,120 @@
+[common]
+"work" = "작업중"
+"break" = "쉬는중"
+"on" = "ON"
+"off" = "OFF"
+"sir" = "{0}님, "
+
+[command]
+"option-warn" = "{0}님, 옵션이 올바르게 설정되어 있는지 확인해주세요." # 0: Username
+"error" = "{0}님 오류가 발생하였습니다. 다시 시도해 주세요" # 0: Username
+"rp-earned" = "（{0} RP）" # 0: Earned
+"not-enter" = "{0}님, 아직 입장하지 않으셨습니다. 「{1}」로 입실해주세요!" # 0: Username, 1: InCommand
+"enter-only" = "{0}씨, 입실중에만 사용할 수 있는 커맨드입니다" # 0: Username  # TODO check
```

Comment:
@gangjun06 
I added some sentences generated by google Translate in `ko.toml`. (see # TODO)
Can you check if these are natural, please?

Reply:
@sorarideblog 
3593f4ee3c2689a461c9b868c6d1f3ff12bed6d3
Yes, I checked.